core/runtime/binaryen/runtime_manager.cpp

```diff
@@ -100,10 +100,8 @@ namespace kagome::runtime::binaryen {
       }
     }
 
-    if (!external_interface_) {
-      external_interface_ = std::make_shared<RuntimeExternalInterface>(
-          extension_factory_, storage_provider_);
-    }
+    external_interface_ = std::make_shared<RuntimeExternalInterface>(
+        extension_factory_, storage_provider_);
 
```

Comment:
It's okay to rawrite it? This member will be rewrited at each call of `createRuntimeEnvironment`. Perhaps would be better to replace it by local variable.

Reply:
It was this way before I refactored RuntimeManager so honestly I've just rolled it back. Will see to it, thank you.